src/all-buildids/Cargo.toml

```
@@ -0,0 +1,17 @@
+[package]
+name = "mz-all-buildids"
```

Since this is going in the `mz-` namespace, can we use the a more generic name instead of "all-buildids"? Maybe we will add other build ID-related functionality in the future!

```suggestion
name = "mz-build-id"
```